html/assets/machinery.js

```diff
@@ -30,6 +30,7 @@ $(document).ready(function () {
   $.each(scopes, function(index, scope) {
     templates[scope] = Hogan.compile($("#scope_" + scope).html());
   });
+  templates["inspection_details_template"] = Hogan.compile($('#inspection_details_template').html());
```

Comment:
Line is too long.<br>['inspection_details_template'] is better written in dot notation.<br>Mixed double and single quotes.